app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -62,6 +62,11 @@ def payload
       }
     end
 
+    def payload_json
+      return @payload_json if defined?(@payload_json)
+      @payload_json = payload.to_json
```

Comment:
I think `.to_json` will never return `nil` so this can be the `||=` syntax:
```suggestion
      @payload_json ||= payload.to_json
```